README.md

```diff
@@ -119,3 +119,16 @@ seneca.act({
 // This is most likely what you will want
 // to be using.
 ```
+
+## testing
+
+By default this libraries maintains a connection to ElasticSearch using a keepAlice.  If you use this library as part of your own plugin,  you'll find that your tests never complete.  To disable the keepAlive, do: 
```

Comment:
The library's own tests aren't setting keepalive, or sniffInterval:0.

Can you give more information maybe?
Also, there is a typo here.